src/handlebars/releases.handlebars

```diff
@@ -37,23 +37,48 @@
       <a id="description_link" href=""></a>
       <h3><a href="./release_notes.html">All Release Notes</a></h3>
 
+      <div class="filter-divs">
+        <div class="filter-os filter-div">
+          <span><strong>Operating System:</strong></span>
+          <select id="os-filter" class="filter" onchange="filterOS()">
+            <option value="Any">-Any-</option>
+            <option value="Linux">Linux</option>
+            <option value="macOS">macOS</option>
+            <option value="AIX">AIX</option>
+            <option value="Windows">Windows</option>
+            <option value="Solaris">Solaris</option>
```

Comment:
I'd have thought the OS list would be dynamically populated with all available OSes for which there is a release.

Reply:
yeah I'll get that implemented today